plugins/modules/fusion_se.py

```diff
@@ -53,23 +53,48 @@
     type: str
     default: iscsi
     choices: [ iscsi ]
+  iscsi:
+    description:
+    - List of discovery interfaces.
+    type: list
+    elements: dict
+    suboptions:
+      address:
+        description:
+        - IP address to be used in the subnet of the storage endpoint.
+        - IP address must include a CIDR notation.
+        - Only IPv4 is supported at the moment.
+        type: str
+      gateway:
+        description:
+        - Address of the subnet gateway.
+        type: str
+      network_interface_groups:
+        description:
+        - List of network interface groups to assign to the address.
+        type: list
+        elements: str
   network_interface_groups:
     description:
+    - "DEPRECATED: Will be removed in version 2.0.0"
```

Comment:
This message should not be in the `description`.  There is  `removed_in_version` for this. See https://docs.ansible.com/ansible/latest/dev_guide/developing_program_flow_modules.html#argument-spec fro the details

Reply:
can not be done